packages/framework/esm-styleguide/src/datepicker/react-spectrum/adobe-react-spectrum-date-wrapper.component.tsx

```diff
@@ -30,11 +31,18 @@ function toLocalDateString(dateValue: Date) {
   return `${year}-${month}-${date}`;
 }
 
-function parseToCalendarDate(date: string | Date | undefined) {
+function parseToCalendarDate(date: string | Date | undefined, locale?: string) {
   if (!date) {
     return undefined;
   }
-  return parseDate(toLocalDateString(typeof date === 'string' ? parseDateString(date) : date));
+
+  const parsedCalendarDate = parseDate(toLocalDateString(typeof date === 'string' ? parseDateString(date) : date));
```

Comment:
This whole construction can be reduced to:

```suggestion
  const parsedCalendarDate = parseDate(dayjs(date).format('YYYY-MM-DD'));
```